Microsoft.Identity.Web.Test/WebApiServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using System.Linq;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class WebApiServiceCollectionExtensionsTests
+    {
+        [Fact]
+        public void TestAuthority()
+        {
+            // Arrange
+            JwtBearerOptions options = new JwtBearerOptions();
```

Comment:
do we need to cover sovereign clouds as well? maybe as a work item?

Reply:
I'm not sure what this changes (the domain would be different). But I added one test.